### DISCUSSION TITLE
GHelper: Custom refresh rate buttons

### DISCUSSION THREAD

**Discussion by Jaythebae98** [seerge/g-helper#2956] [Ideas]
@seerge Hey dude , love the product for the last year , I was wondering is there a way to customise extra buttons on the refresh rates, like my on battery defaults to 60 Hz but I like to use it at 100Hz on battery and 144Hz on power but it reverts back to 60 so I have to go into additional settings on display in settings each time , I was wondering if this is possible or even if its in the works?
I have the Zephyrus g14 GA402RK 2022

**Comment by seerge** [comment]
@Jaythebae98 hello, I don't think your device even supports 100Hz. You (most likely) have 2 options in windows display settings, 60hz and 120hz (or 144hz depending on a panel type). App just gives you a shortcut to those windows settings.

**Reply by NJinsei** [reply]
The poster could be referring to using CRU where a custom refresh rate can be set (aka 100Hz) and then be asking to use ghelper to switch between with the custom refresh rate, instead of only being able to use the original refresh rate values.

**Reply by seerge** [reply]
I don't think it appears in windows settings anyway after those edits :) so you need to rely on CRU.

**Comment by TylerSmith72** [comment]
I also like this idea. My laptop (Zephyrus G15 GA503RW) can do up to 240Hz but when I use 60Hz it doesn't feel smooth. I know that 60Hz won't be as good as 240Hz but it feels like it stutters/drops to 40 or 50Hz sometimes. 

Windows allows me to set these values:
![image](https://github.com/user-attachments/assets/664d44dc-eb93-4614-af37-989ae8c9a1be)
but it would help if these options were in G-Helper too

Would it be possible to add other screen Hz values to G-Helper and have it work with the 'Auto' mode please? So that we can set a lower value and a higher value. For example, 120Hz and 240Hz and it will switch automatically when on battery/plugged in. Like it already does except we're not locked to either 60Hz or our maximum Hz
